### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sebmartin**
This sucks

### CODE REVIEW

**Comment by sebmartin** [comment]
....................../´¯/) 
....................,/¯../ 
.................../..../ 
............./´¯/'...'/´¯¯`·¸ 
........../'/.../..../......./¨¯\ 
........('(...´...´.... ¯~/'...') 
.........\.................'...../ 
..........''...\.......... _.·´ 
............\..............( 
..............\.............\...

**Comment by sebmartin** [comment]
ping @etiennemartin

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # CodeReviewTesting
 This is only a testing repo used to play with GitHub's APIs
+
+This repo fucking sucks
```
Comment (etiennemartin) [inline README.md:4]: This is perfect.  Don't change anything.

**Review by etiennemartin — CHANGES_REQUESTED** [review]
This is amazing.  You're so talented.  I wish I was as good at writing README's as you are.  You're my hero.

**Pull request closed without merging**